lib/DBus/Client.hs

```diff
@@ -154,21 +158,28 @@ module DBus.Client
     , defaultClientOptions
     , connectWith
 
+    , dbusName
+    , dbusPath
+
+    , ErrorName
+    , errorFailed
     , errorInvalidParameters
+    , errorUnknownMethod
     ) where
 
-import Control.Concurrent
-
 import Control.Applicative
 import Control.Arrow
+import Control.Concurrent
 import qualified Control.Exception
 import Control.Exception (SomeException, throwIO)
 import Control.Lens
 import Control.Monad
+import Control.Monad.Reader
```

Comment:
Can't we use Reader from transformers? Using mtl and transformers in the same library looks counterintuitive.

Reply:
yep. good catch.